config/config.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
@lenon777, you have put the config folder to gitignore, but config.json still here. Is it planned action?